src/Components/HomeCards/HomeCards.tsx

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import styles from "./HomeCards.module.css";
+
+function HomeCards() {
+  return (
+    <article className={styles.cards}>
+      <div className={styles.singlecard}>
+        <h4 className={styles.categories}>outdoor schedule</h4>
+        <svg
+          className={styles.icons}
+          width="70"
+          height="70"
+          fill="none"
+          viewBox="0 0 36 36"
+        >
+          <g transform="translate(0 -1020.362)">
+            <path d="m 15.99903,1028.8625 c -4.13591,0 -7.49945,3.3638 -7.49945,7.5001 0,4.1363 3.36354,7.5002 7.49945,7.5002 4.1359,0 7.49944,-3.3639 7.49944,-7.5002 0,-4.1363 -3.36354,-7.5001 -7.49944,-7.5001 z m 0,1 c 3.5955,0 6.49952,2.9042 6.49952,6.5001 0,3.5959 -2.90402,6.5002 -6.49952,6.5002 -3.59551,0 -6.49952,-2.9043 -6.49952,-6.5002 0,-3.5959 2.90401,-6.5001 6.49952,-6.5001 z" />
+            <path d="M15.99122 1022.3623a.50001284.5000613 0 00-.49216.5078l0 2.9845a.50001284.5000613 0 10.99993 0l0-2.9845a.50001284.5000613 0 00-.50777-.5078zM15.99122 1046.3628a.50001284.5000613 0 00-.49216.5079l0 2.9844a.50001284.5000613 0 10.99993 0l0-2.9844a.50001284.5000613 0 00-.50777-.5079zM2.50784 1035.8626a.50001284.5000613 0 100 1l2.9861 0a.50001284.5000613 0 100-1zM26.50606 1035.8626a.50001284.5000613 0 100 1l2.9861 0a.50001284.5000613 0 100-1zM25.48466 1026.3624a.50001284.5000613 0 00-.34373.1523l-1.9901 1.9883a.50001284.5000613 0 10.70698.7072l1.9901-1.9884a.50001284.5000613 0 00-.36325-.8594zM6.49973 1026.3624a.50001284.5000613 0 00-.34763.8594l1.99009 1.9884a.50001284.5000613 0 10.70698-.7072l-1.99009-1.9883a.50001284.5000613 0 00-.35935-.1523zM23.49847 1043.3628a.50001284.5000613 0 00-.34764.8594l1.9901 1.9883a.50001284.5000613 0 10.70698-.7071l-1.9901-1.9883a.50001284.5000613 0 00-.35934-.1523zM8.48591 1043.3628a.50001284.5000613 0 00-.34372.1523l-1.99009 1.9883a.50001284.5000613 0 10.70698.7071l1.99009-1.9883a.50001284.5000613 0 00-.36326-.8594z" />
+          </g>
+        </svg>
+      </div>
```

Comment:
Not necessarily needed, but maybe a generic `Card` component would also make sense her :)